htdocs/core/tpl/objectline_title.tpl.php

```diff
@@ -56,7 +56,23 @@
 }
 
 // Description
-print '<th class="linecoldescription">'.$langs->trans('Description').'</th>';
+print '<th class="linecoldescription">'.$langs->trans('Description');
+if (in_array($object->element, array('propal', 'commande', 'facture', 'order_supplier', 'invoice_supplier')) && $object->status == $object::STATUS_DRAFT) {
```

Comment:
The problem is due to the line 
@phan-var-force CommonObject
few line before. It says to phan than $object is CommonObject but it is not, it is Facture or Propal or Commande...

Can you try this workaround:
`$constant = get_class($object)."::STATUS_DRAFT";
if (in_array($object->element, array('propal', 'commande', 'facture', 'order_supplier', 'invoice_supplier')) && defined($constant) && $object->status == constant($constant)) {
`